### DISCUSSION TITLE
Multiple functions

### DISCUSSION THREAD

**Discussion by LastSpirit** [jaredpalmer/formik#3573] [Q&A]
Good afternoon,
How do I use multiple functions in onBlur?
React Native
[
![Screenshot_1](https://user-images.githubusercontent.com/61553803/169688802-83c5faef-d67c-4ea3-8352-7840aebcc749.jpg)
](url)

**Comment by carlos-dubon** [comment]
Just use another function to wrap the 2 functions you want to call.

```JavaScript
onBlur={handleMyBlur}

handleMyBlur = () => {
    handleBlur();
    setIsFocused(true);
}
```
